src/expressions/dataTypes/documentOrderUtils.ts

```diff
@@ -25,6 +25,7 @@ function compareSiblingElements(
 	node2: NodePointer
 ): number {
 	if (arePointersEqual(node1, node2)) {
+		/* istanbul ignore next */
```

Comment:
I talked about this with Mehmet and it seems this is dead code for now. I understood we want to keep these checks in here in case we want to re-use this function elsewhere which I'm ok with.

But if we do add code elsewhere which suddenly cause these lines to no longer be dead code than we'd also have to remove these ignores which I doubt will happen as it's hard to keep track of.

So I'm ok with having some dead code to make functions more future proof but than we should also accept that the lines are not covered for now.

Reply:
Or we could remove the dead code as we do for `compareElements` a bit below.

Reply:
I am going to remove the dead code. If we need it in the future, we can add it.